packages/superset-ui-time-format/src/utils/createTimeRangeFromGranularity.ts

```diff
@@ -0,0 +1,94 @@
+import { TimeGranularity } from '../types';
+import createDate from './createDate';
+
+const MS_IN_SECOND = 1000;
+const MS_IN_MINUTE = 60 * MS_IN_SECOND;
+const MS_IN_HOUR = 60 * MS_IN_MINUTE;
+const MS_IN_DAY = 24 * MS_IN_HOUR;
+const MS_IN_WEEK = 7 * MS_IN_DAY;
```

Comment:
Can probably remove `MS_IN_` to make it easier to read?

Reply:
To avoid confusion with `TimeGranularity.HOUR`, etc. Also make it clear that the value is milliseconds.